### PR TITLE
Add table property documentataion and release notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-docs/src/main/sphinx/sql/create-table-as.rst [hunk 1]
```diff
@@ -15,12 +17,21 @@ Description
 Create a new table containing the result of a :doc:`select` query.
 Use :doc:`create-table` to create an empty table.
 
+The optional ``WITH`` cause can be used to set properties
```
Comment (martint) [inline presto-docs/src/main/sphinx/sql/create-table-as.rst]: cause -> clause?

**Comment by electrum** [comment]
Document the new system table in `system.rst`

File: presto-docs/src/main/sphinx/sql/create-table-as.rst [hunk 2]
```diff
@@ -7,7 +7,9 @@ Synopsis
 
 .. code-block:: none
 
-    CREATE TABLE table_name AS query
+    CREATE TABLE table_name
+    [WITH (property_name = expression [, ...])]
```
Comment (electrum) [inline presto-docs/src/main/sphinx/sql/create-table-as.rst]: Add spaces around the brackets. For example, like `select.rst`

Comment (electrum) [inline presto-docs/src/main/sphinx/sql/create-table-as.rst]: Add spaces around the brackets and parens. For example, see `select.rst`

```
[ WITH ( property_name = expression [, ...] ) ]

File: presto-docs/src/main/sphinx/sql/create-table.rst [hunk 3]
```diff
@@ -7,9 +7,12 @@ Synopsis
 
 .. code-block:: none
 
-    CREATE TABLE [ IF NOT EXISTS ] table_name (
+    CREATE TABLE [ IF NOT EXISTS ]
+    table_name (
       column_name data_type [, ...]
     )
+    [WITH (property_name = expression [, ...])]
```
Comment (electrum) [inline presto-docs/src/main/sphinx/sql/create-table.rst]: Same

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 4]
```diff
@@ -34,6 +34,11 @@ System Connector Tables
 
 The catalogs table contains the list of available catalogs.
 
+``metadata.table_properties``
+^^^^^^^^^^^^^^^^^^^^
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/system.rst]: This needs to be the same length as the text

I think the Python IDEA plugin will warn

File: presto-docs/src/main/sphinx/connector/system.rst [hunk 5]
```diff
@@ -34,6 +34,11 @@ System Connector Tables
...
+
+The table_properties table contains the list of available table properties.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/system.rst]: The table properties table contains the list of available table properties that can be set for tables in a connector.

File: presto-docs/src/main/sphinx/release/release-0.115.rst [hunk 6]
```diff
@@ -2,6 +2,17 @@
...
 
+Table Properties
+----------------
+
+When creating tables with :doc:`/sql/create-table` or :doc:`/sql/create-table-as`,
+you can now add connector specific properties to the new table.  For example, when
+creating a Hive table you can specify the file format.  To list all available table
+properties, execute the following query::
+
+    SELECT *
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.115.rst]: This is easier to read on one line

File: presto-docs/src/main/sphinx/release/release-0.115.rst [hunk 7]
```diff
@@ -2,6 +2,17 @@
 Release 0.115
 =============
 
+Table Properties
+----------------
+
+When creating tables with :doc:`/sql/create-table` or :doc:`/sql/create-table-as`,
+you can now add connector specific properties to the new table.  For example, when
+creating a Hive table you can specify the file format.  To list all available table
+properties, execute the following query::
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.115.rst]: execute -> run

**Comment by electrum** [comment]
Looks good

**Pull request merged**